### ISSUE TITLE
Automatic pre-translation with DeepL

### ISSUE THREAD

**Issue by JSEHV** [inkstitch/inkstitch#1706]
Can you set up the [Automated Pre-translation via MT](https://support.crowdin.com/pre-translation-via-machine/#automated-pre-translation-via-mt) for the translations. 

This way we only need to review, adjust and approve and not copy-paste all the base translations. 
DeepL is the most accurate, so this would be preferred compared to other services.

**Comment by JSEHV** [comment]
Fair point :)

**Comment by lexelby** [comment]
@JSEHV By the way, if you have put in automated translations for other languages, could you please go and remove them to avoid the issues mentioned above?

**Comment by kaalleen** [comment]
I think we can handle that in most parts with the undo option in crowdin. No need for @JSEHV to go through that by hand.

@JSEHV you have been very active and seem to be willing to help. Actually we could need some help in translations. What do you think if we setup a dutch version for our website?

**Comment by JSEHV** [comment]
Only Swedish I have done with DeepL, the other (Czech) is done by my wife, as she is a Slovak native, fluent in Czech too. If you can add a Slovak language we are able to translate both Czech and Slovak.

**Comment by JSEHV** [comment]
> I think we can handle that in most parts with the undo option in crowdin. No need for @JSEHV to go through that by hand.
> 
> @JSEHV you have been very active and seem to be willing to help. Actually we could need some help in translations. What do you think if we setup a dutch version for our website?

Yes, that would be great! I will translate the full website to Dutch.

**Comment by kaalleen** [comment]
Wow thanks!

I'll add a dutch to the website then, but it will take a little while until I will have the time to do so.

**Comment by kaalleen** [comment]
Btw. I changed your role in crowdin so you can approve dutch and czech translations yourself.

I'm not sure how to add a language @lexelby ?

**Comment by JSEHV** [comment]
> Wow thanks!
> 
> I'll add dutch to the website then, but it will take a little while until I will have the time to do so.

If you are ok with accepting a pull request, I can add all the dutch translations (as they are in the same logic as the other languages, most in their own folder, and a dutch index) into the pull request. Will that work for you?

In the same way we can add Czech and/or Slovak later too, but Dutch will be first done :)

**Comment by kaalleen** [comment]
Yes sure. Go ahead!
